backend/README.MD

```
@@ -65,3 +65,19 @@ Follow these instructions to run the FastAPI project:
   pip install <package-name>
   pip freeze > requirements.txt
   ```
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)